aten/src/ATen/native/cuda/DistanceKernel.cu

```diff
@@ -106,53 +106,50 @@ __global__ static void pdist_kernel_cuda_impl(scalar_t * result, const scalar_t
   // This shared memory is significantly larger than necessary, but the
   // assumption is that it's not a bottleneck, and this is simple
   __shared__ scalar_t shared[forward_threads];
-  int lane = threadIdx.y % warpSize;
-  int warp_id = threadIdx.y / warpSize;
+  int lane = threadIdx.x % warpSize;
```

Comment:
what's the reason for using threadIdx.x instead of .y here?

Reply:
Came from the revert.